js/scenes/Surveys/Pages/Surveys.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ScrollView, Text, View } from 'react-native';
+import { GoalsModal, RadioAnswers, WarningModal } from '../SurveyComponents/SurveysComponents';
+import ProgressBar from '../../../components/ProgressBar';
+import LeftNavArrow from '../../../components/LeftNavArrow';
+import RightNavArrow from '../../../components/RightNavArrow';
+import { styles } from '../styles';
+
+const Surveys = ({
+  dispatchAnswer,
+  displayGoalModal,
+  closeWarningModal,
+  goalModalState,
+  warningModalState,
+  question,
+  questions,
+  nextQuestion,
+  previousQuestionFunc,
+  prevQuestion,
+  selectedValue,
+  speakerGoals
+}) => {
+  return (
+    <ScrollView style={styles.sceneContain}>
+      <View style={styles.background}>
+        {(question) ?
```

Comment:
If it's a 50/50 logic use ` or = ||`  or `and = &&`

Reply:
Done